spring-cloud-sleuth-autoconfigure/src/main/java/org/springframework/cloud/sleuth/autoconfig/instrument/jdbc/DataSourceProxyConfiguration.java

```diff
@@ -84,6 +84,8 @@ private DataSourceProxyProperties props(TraceJdbcProperties traceJdbcProperties)
 		DataSourceProxyProperties props = new DataSourceProxyProperties();
 		BeanUtils.copyProperties(originalProxy, props);
 		props.setLogging(DataSourceProxyProperties.DataSourceProxyLogging.valueOf(originalProxy.getLogging().name()));
+		BeanUtils.copyProperties(originalProxy.getQuery(), props.getQuery());
```

Comment:
:grimacing: oops

Reply:
Is this correct approach ? or we can have more optimal way to do it ?

Reply:
I said oops cause I've forgotten to add those lines :facepalm: Great that you caught it!